deploy/install.sh

```diff
@@ -51,7 +51,7 @@ check_os() {
     os_name="$(cat /etc/*-release | awk -F= '$1 == "NAME" { gsub(/"/, ""); print $2; exit }')"
 
     case "$os_name" in
-        Ubuntu*)
+        Ubuntu*|Pop!_OS)
```

Comment:
It is recommended to use double pipe (||) operator for conditions.

Reply:
[Bash manual](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-case) shows that patterns in a list of `case` statement should be separated with a single pipe.

Reply:
Understood. Thanks for the reference.